### ISSUE TITLE
Product image can't display based on variant asset

### ISSUE THREAD

**Issue by blackyzero** [vendure-ecommerce/storefront-remix-starter#11]
Hello,

How could i make product image display based on variant selection ?
For now. it's only show main product image.

Ex: i add image for each item variant
![Screen Shot 2022-06-13 at 9 52 38 AM](https://user-images.githubusercontent.com/4286063/173369659-846f985b-4009-4efe-93c5-e7e6f0c28425.png)

But it doesn't pickup the item variant image from UI.
![Screen Shot 2022-06-13 at 9 57 11 AM](https://user-images.githubusercontent.com/4286063/173370180-6a3346a0-3dbc-4e10-bb3b-36b3e559b02b.png)

Thank you.

**Comment by michaelbromley** [comment]
Yes this is a feature that should be added. I've added some labels to encourage community contribution on this.

**Comment by kyunal** [comment]
I'm gonna give it a shot, just mentioning it in case somebody else is already on it or planning to do it :stuck_out_tongue:

**Comment by blackyzero** [comment]
Great. Thanks for adding this. It's definitely useful and increasing one more cool thing of existing remix storefront.

**Comment by kyunal** [comment]
Closed as this was solved in https://github.com/vendure-ecommerce/storefront-remix-starter/pull/13
